app/migrations.go

```diff
@@ -156,3 +157,56 @@ func (a *App) DoEmojisPermissionsMigration() {
 		mlog.Critical(fmt.Sprint(result.Err))
 	}
 }
+
+func (a *App) DoGuestRolesCreationMigration() {
```

Comment:
It is a data migration and I'm using all the `Save` store function logic, I'm not 100% sure that I can use store functions during the upgrade. I think I can use the SQL connection, but not the Store itself.

Reply:
I think I'm wrong, I going to try to move it to the upgrade and see what happen

Reply:
Ok, now I know exactly what is going on, because we are using the "layered" approach for roles and schemes, this aren't available until after the upgrade, so... is not easy to use the store functions to make the migration there.

Reply:
Ahhhhh of course. We really need to fix that design flaw!

Reply:
But not in this PR ;) I'm happy for this to stay as an app-layer migration.